Refactor threshold trigger and condition docs into shared includes (#45719)
home-assistant/home-assistant.github.io: source/_includes/conditions/threshold_value_options_ui.md

{% comment %}
Reusable "Options in the UI" block for entity conditions that test a reading
against the threshold-mapping schema.

Parameters:

>   reading            quantity noun, e.g. "humidity"

Reviewer: ```suggestion
  reading            Quantity noun. For example: "humidity".
```
What do you mean by a quantity noun?

Author: its the noun of the reading. maybe "Reading type" would be better?

Reviewer: as in: domain?

Reviewer: I think we could just remove that variable. The text works without it. One less potential source of copy-paste errors.

>   value_short        Number-entry phrasing, e.g. "a fixed percentage (0–100)"

Reviewer: ```suggestion
  value_short        Number-entry phrasing. For example: "a fixed percentage (0–100)".
```

  has_unit           set (to anything) to add a "Unit" row (requires unit_label,
                     unit_options_code, unit_default)

>   unit_label         unit-kind label, e.g. "temperature unit", "energy unit"

Reviewer: ```suggestion
  unit_label         Unit label. For example: "temperature unit", "energy unit".
```

Author: "Unit type" can be confused with the "reading (quantity noun)"

>   unit_options_code  inline-code list of allowed units, e.g. "`°C` or `°F`"

Reviewer: ```suggestion
  unit_options_code  Inline code list of allowed units. For example: "`°C` or `°F`".
```

Author: these are comments for the include, they are not visible in the resulting page. should I update them all to add leading period and start with Caps?

Reviewer: yes please

>   unit_default       default unit, e.g. "°C"

Reviewer: ```suggestion
  unit_default       Default unit. For example: "°C"
```

{% endcomment %}
{% options_ui %}
Threshold type:
  description: |

>     The {{ include.reading }} level the entity has to meet for the condition to pass. **Above** and **Below** are exclusive: a reading equal to the threshold does not pass. **In range** is exclusive at both bounds. **Outside range** is inclusive: a reading equal to either bound passes. Choose **Number** to enter {{ include.value_short }}, or **Entity** to use a sensor or number helper as a dynamic threshold.

Reviewer: ```suggestion
    The level the entity has to meet for the condition to pass. **Above** and **Below** are exclusive: a reading equal to the threshold does not pass. **In range** is exclusive at both bounds. **Outside range** is inclusive: a reading equal to either bound passes. Choose **Number** to enter {{ include.value_short }}, or **Entity** to use a sensor or number helper as a dynamic threshold.
```
maybe we can just remove this to make the text more generic.
